content/docs/v2/theming/css-utilities/index.md

```diff
@@ -66,9 +66,15 @@ Ionic provides a set of SCSS/CSS utility attributes that can help when developin
 
 | Attribute | Description                                                                                                |
 |-----------|------------------------------------------------------------------------------------------------------------|
-| `start`   | Aligns the element at the start based on platform. iOS goes left, Android is the first item on the right   |
-| `end`     | Aligns the element at the end based on the platform. iOS goes right, Android is the last item on the right |
-| `left`    | Aligns the element on the left side. Overrides the platform specific order                                 |
-| `right`   | Aligns the element on the right side. Overrides the platform specific order                                |
+| `start`   | aligns the element at the start based on platform. iOS goes left, Android is the first item on the right   |
+| `end`     | aligns the element at the end based on the platform. iOS goes right, Android is the last item on the right |
+| `left`    | aligns the element on the left side. Overrides the platform specific order                                 |
+| `right`   | aligns the element on the right side. Overrides the platform specific order                                |
 
 
+### `ion-checkbox, ion-select, ion-toggle` property
```

Comment:
Isn't this already covered by 
https://github.com/driftyco/ionic-site/blob/master/content/docs/v2/theming/css-utilities/index.md#text-transformation

If not, we could just add a note there instead to cut down on repeat content.